ts/src/bitflyer.ts

```diff
@@ -955,9 +956,9 @@ export default class bitflyer extends Exchange {
         if ('fee' in transaction) {
             type = 'withdrawal';
             status = this.parseWithdrawalStatus (rawStatus);
-            const feeCost = this.safeNumber (transaction, 'fee');
-            const additionalFee = this.safeNumber (transaction, 'additional_fee');
-            fee = { 'currency': code, 'cost': feeCost + additionalFee };
+            const feeCost = this.safeString (transaction, 'fee');
+            const additionalFee = this.safeString (transaction, 'additional_fee');
+            fee = { 'currency': code, 'cost': Precise.stringAdd (feeCost, additionalFee) };
```

Comment:
@samgermain yeah but we can't return as a string, we have to pass it through `parseNumber`